web/src/components/MapView.tsx

```diff
@@ -28,19 +34,92 @@ const layerStyle: LayerProps = {
   paint: {}
 }
 
+const renderPopup = (coordinates: Position, description: string): ReactElement => {
+  return (
+    <StyledPopup
+      longitude={coordinates[0]}
+      latitude={coordinates[1]}
+      closeButton={false}
+      closeOnClick={false}
+      anchor='bottom'>
+      <div dangerouslySetInnerHTML={{ __html: description }} />
+    </StyledPopup>
+  )
+}
+
+const getPopUpDescription = (properties: GeoJsonProperties): string => {
+  return `<div><strong>${properties?.title}</strong></div>`
+}
+
+const getCursor = ({ isHovering, isDragging }: { isHovering: boolean; isDragging: boolean }): Property.Cursor => {
```

Comment:
`Property.Cursor` is the same as `React.CSSProperties['cursor']`
I don't think we need the library here...

Reply:
well i think its not as good as the csstype lib since the typing is quite poor. CSSType offers way more exact types f.e. unions instead of strings. Since its a dev dependency it won't increase any performance or package size. We can discuss that tommorow :)